2.5.0/spec_sp/coorder.md

```diff
@@ -32,7 +32,7 @@ end
 ## Request
 
 ```http
-POST coorder/ HTTP/1.1
+POST coorder/2.5/ HTTP/1.1
```

Comment:
Shouldn't the version go before the endpoint name? Also please add the leading slash so it's valid http.
```http
POST /onapi/2.5/coorder
```